Dockerfile

```diff
@@ -3,26 +3,21 @@ ARG NODE_IMAGE_VERSION=13.5.0-alpine
 FROM node:$NODE_IMAGE_VERSION as builder
 
 COPY package*.json ./
-
 RUN npm install
 
 COPY . .
 
 RUN npm run compile
-
-RUN npm run generateDocs
-
 RUN npm test
+RUN npm run generateDocs
 
 FROM node:$NODE_IMAGE_VERSION
 
 COPY --from=builder dist dist
-COPY package*.json ./
-COPY docs /docs
-
-RUN npm install --production
-
-COPY docs /docs
+COPY --from=builder node_modules node_modules
```

Comment:
If you do this it's useless to have multiple stages.
And you are copying even the devDependencies

Reply:
You're right. Added the `npm install --production` and removed this COPY.